frontend/src/scenes/onboarding/sdks/product-analytics/android.tsx

```diff
@@ -16,29 +17,36 @@ function AndroidCaptureSnippet(): JSX.Element {
 
 function AdvertiseAndroidReplay(): JSX.Element {
     return (
-        <div>
-            <h3 className="mt-8">
-                Session Replay for Android <LemonTag type="highlight">NEW</LemonTag>
-            </h3>
-            <div>
-                Session replay is now in beta for Android.{' '}
-                <Link to={urls.onboarding('session_replay', OnboardingStepKey.INSTALL, SDKKey.ANDROID)}>
-                    Learn how to set it up
-                </Link>
-            </div>
-        </div>
+        <>
+            <LemonBanner type="info" className="mt-8">
+                <div>
+                    <h3>
+                        Session Replay for Android <LemonTag type="highlight">NEW</LemonTag>
+                    </h3>
+                    <div>
+                        Session replay is now in beta for Android.{' '}
+                        <Link
+                            to={urls.onboarding('session_replay', OnboardingStepKey.INSTALL, SDKKey.ANDROID)}
+                            data-attr="product-analytics-onboarding-android-replay-cta"
+                        >
+                            Learn how to set it up
+                        </Link>
+                    </div>
+                </div>
+            </LemonBanner>
+        </>
     )
 }
 
 export function ProductAnalyticsAndroidInstructions(): JSX.Element {
     return (
         <>
-            <SDKInstallAndroidInstructions />
-            <h3>Send an Event</h3>
-            <AndroidCaptureSnippet />
             <FlaggedFeature flag={FEATURE_FLAGS.SESSION_REPLAY_MOBILE_ONBOARDING} match={true}>
                 <AdvertiseAndroidReplay />
             </FlaggedFeature>
+            <SDKInstallAndroidInstructions />
```

Comment:
To me, it feels strange that we offer a feature before having the SDK installed, and the feature requires the SDK installed first.
So the current sorting feels more natural, I install the SDK and then I try out something new

Reply:
Yeah, however I changed things the spacing looked wrong... I'll have another try first